features/setup.py

```diff
@@ -21,6 +21,7 @@
     'numpy',
     'pandas',
     'scikit-learn',
+    'holidays',
```

Comment:
We want to be extremely selective with which packages we install by default in `autogluon.features`. We probably want to move holidays to be an optional dependency not installed by default, with the holidays logic not enabled by default.